Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/AssetGroups/AssetAssetGroup.tsx

```diff
@@ -166,7 +166,7 @@ function AssetAssetGroupWindow(props: { AssetGroupID: number}) {
                 </div>
             </div>
             </div>
-            <AssetSelect Type='multiple' SessionStorageID='AssetAssetGroup' Title='Add Transmission Assets to Asset Group' ShowModal={showAdd} SelectedAssets={assetList}
+            <AssetSelect Type='multiple' StorageID='AssetAssetGroup' Title='Add Transmission Assets to Asset Group' ShowModal={showAdd} SelectedAssets={assetList}
```

Comment:
I am a little confused why we renamed this in the latest `gpa-gemstone` commit? I thought this was an addition?

Reply:
It is, but session storage is a little confusing since it implies we were saving it in the browsers session storage, which isn't true, we were saving it local storage, which is persistent across refreshes.

Reply:
I believe I added that argument in preparation for this change since I knew the change was coming. It wasn't hooked into anything until this update